docs/docs/component-scatterplot.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Should we do other components?  I can help out with this effort.

Reply:
Potentially but I think before we do that we should try out this example in a fresh create-react-app and confirm that it actually works - it has been a while since I tried that